exercises/react/example.cr

```diff
@@ -0,0 +1,71 @@
+alias CellValue = Int32
+
+private class Cell
+  @dependents = [] of ComputeCell
+
+  def initialize(initial : CellValue)
+    @value = initial
+  end
+
+  getter :value
+
+  protected getter :dependents
+  protected def propagate; _propagate; end
```

Comment:
@petertseng I believe this is a bug, I just created an issue for it: https://github.com/crystal-lang/crystal/issues/3290

Whenever you find something that looks suspicious, don't hesitate to report it. The language isn't 1.0 yet so there are still some bugs and corner cases. Thank you!